pandas/core/window/rolling.py

```diff
@@ -536,6 +545,8 @@ def _apply(
         use_numba_cache : bool
             whether to cache a numba compiled function. Only available for numba
             enabled methods (so far only apply)
+        skip_offset : bool
```

Comment:
what's the point of an addtional parameter here? this makes it really hard to understand

Reply:
@justinessert can you address